db/schema.rb

```diff
@@ -55,6 +55,17 @@
     t.index ["user_id"], name: "index_authorizations_on_user_id"
   end
 
+  create_table "backup_code_configurations", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.string "encrypted_code", default: "", null: false
+    t.string "code_fingerprint", default: "", null: false
+    t.boolean "used", default: false
+    t.datetime "used_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id", "code_fingerprint"], name: "index_bcc_on_user_id_code_fingerprint"
```

Comment:
The migration wants this to be unique, but the schema file doesn't have it marked as unique. Might need to rerun the migration.